### PR TITLE
Implement event statistic API on event overview

### LINKED ISSUE #636: Implement Event statistics API on event overview page
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend
image

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #636

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-679-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-679-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-679-open-event-frontend-fossasia.surge.sh

File: app/templates/components/events/view/overview/general-info.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="header">{{t 'General info'}}</div>
 </div>
 <div class="content">
-  <div class="header"> {{t 'Your event is not live yet. Make it live now'}}</div>
+  <div class="header"> {{t 'Your event is not live yet.  Make it live now'}}</div>
```
Comment (geekyd) [inline app/templates/components/events/view/overview/general-info.hbs]: @CosmicCoder96 This should be dynamic(not live/ live). Use `event.state` for this.

File: app/templates/components/events/view/overview/general-info.hbs [hunk 2]
```diff
@@ -13,35 +13,43 @@
       </tr>
       <tr>
         <td><strong>{{t 'Location'}}</strong></td>
-        <td>asdad</td>
+        <td>{{data.event.locationName}}</td>
       </tr>
       <tr>
         <td><strong>{{t 'Start time'}}</strong></td>
-        <td>12:30 PM May 11, 2014</td>
+        <td>{{moment-format data.event.startsAt 'MMMM Do YYYY, h:mm:ss A'}}</td>
```
Comment (geekyd) [inline app/templates/components/events/view/overview/general-info.hbs]: Remove the seconds from all the dates. Also the format seems to differ from the desired output.

Comment (CosmicCoder96) [inline app/templates/components/events/view/overview/general-info.hbs]: What's the desired output?

Comment (geekyd) [inline app/templates/components/events/view/overview/general-info.hbs]: I think `12:30 PM May 11, 2014` right?

**Comment by CosmicCoder96** [comment]
@geekyd please review

File: app/templates/components/events/view/overview/general-info.hbs (hunk 1 above)
Comment (CosmicCoder96) [inline app/templates/components/events/view/overview/general-info.hbs]: done

**Comment by niranjan94** [comment]
Deployment link: https://pr-679-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-679-open-event-frontend-fossasia.surge.sh

**Pull request merged**